bcda/client/bluebutton.go

```diff
@@ -333,8 +333,8 @@ func addRequestHeaders(req *http.Request, reqID uuid.UUID, jobID, cmsID string)
 	req.Header.Add("keep-alive", "")
 	req.Header.Add("BlueButton-OriginalUrl", req.URL.String())
 	req.Header.Add("BlueButton-OriginalQuery", req.URL.RawQuery)
-	req.Header.Add("BCDA-JOBID", jobID)
-	req.Header.Add("BCDA-CMSID", cmsID)
+	req.Header.Add("BULK-JOBID", jobID)
```

Comment:
Would it make sense for us to pull out these headers as constants? I don't think we'll need it exposed beyond the package.

So maybe something like
```
const (
jobIDHeader = "BULK-JOBID"
clientIDHeader = "BULK-CLIENTID"
)
```

Reference: https://golang.org/doc/effective_go.html#mixed-caps